tests/vm

```diff
@@ -21,15 +21,20 @@ if ! echo "${LXD_SNAP_CHANNEL}" | grep -qE '^4\.0/'; then
     lxc start vm1
     waitInstanceBooted vm1
 
-    echo "==> Test lxc exec exit code when stopping the VM cleanly"
-    (sleep 5 && lxc stop vm1) &
-    lxc exec vm1 -- sleep 60 || exitCode=$?
-    [ "${exitCode:-0}" -eq 129 ]
-    wait $!
+    echo "==> Test getting the correct exit code for a signaled process"
+    # Signaling the process spawned by lxc exec and checking its exit code.
+    # Simulates what can happen if the container stops in the middle of lxc exec.
```

Comment:
"container" in the `tests/vm` suite?

Reply:
whoops, sorry about that. My local version is correct but I forgot to push this change :woozy_face: